### PR TITLE
Proposal: _always_ include default released feature flags

### PR DESCRIPTION
**Pull request by macobo**
This caused pain for a client who we did a beta release for.

### CODE REVIEW

File: posthog/settings.py [hunk 1]
```diff
@@ -114,16 +114,18 @@ def get_list(text: str) -> List[str]:
...
+PERSISTED_FEATURE_FLAGS = [
+    # Add hard-coded feature flags for static releases here
+    "3638-trailing-wau-mau",  # pending UI/UX improvements; functionality ready
+    "5440-multivariate-support",
+    "4141-event-columns",
+    "new-paths-ui",
+    "new-paths-ui-edge-weights",
+]
+
 PERSISTED_FEATURE_FLAGS = []
```
Comment (Twixes) [inline posthog/settings.py:126]: I suppose the latter `PERSISTED_FEATURE_FLAGS` shouldn't be here?

**Review by Twixes — COMMENTED** [review]
As far as I understand, this will also fully release these `PERSISTED_FEATURE_FLAGS` on Cloud (which currently uses the special env var value `0`), and out of 5 these flags, only 1 is currently fully rolled out on Cloud. So I'm 100% for this _if_ we do it in a way that keeps `PERSISTED_FEATURE_FLAGS` empty on Cloud to avoid disrupting things for now.

**Comment by macobo** [comment]
Why are we releasing features on self-hosted and not in cloud? :thinking: 

All of this kind of sounds like our feature release process is smelly and needs simplifications to keep the release and cloud released features in sync. After all, we expect the logic if "something is released" to be a simple question.

The solution can be this PR or it can be deleting feature flags in a more timely manner. /shrug

**Review by mariusandra — APPROVED** [review]
LGTM. [We've had issues](https://github.com/PostHog/posthog/pull/5296#issuecomment-899125238) with this already before.

**Comment by Twixes** [comment]
I totally agree @macobo Just that we should do the full simplification separately from this PR. CC @paolodamico

**Review by Twixes — CHANGES_REQUESTED** [review]
Also just noticed that, as noted in `# These flags will be force-enabled on the frontend **and OVERRIDE all** flags from decide`, as-is this change causes all flags that aren't in `PERSISTED_FEATURE_FLAGS` to be disabled, which force-disables all our WIP or Cloud-only features (e.g. groups, the announcement bar).

**Review by paolodamico — APPROVED** [review]
I don't think this is the right approach. Here are some arguments as to the benefits of cloud and self-hosted **not** being in sync:
1. On Cloud we run A/B tests aside from phased roll-outs to test improvements and no regressions. We would essentially lose our ability to A/B test by doing this.
2. Cloud is often used as a sandbox for phased roll-outs, we test new features / improvements with a % of Cloud users, and if they work we include them in a self-hosted release. The reason for this is that we can correct course in Cloud much faster if something goes wrong.
3. We have other functionality on Cloud that relies on FF for logic (e.g. in-product announcements)

-------

Based on what prompted this (i.e. the problem we're trying to solve is for self-hosted users being able to easily enable alpha/beta features for their instance), I've pushed a counter proposal. We solve the problem that prompted this, but not what @mariusandra brings up, we will still need to have good stewardship in FF management.

-------

Finally, the proposal also fixes for using FFs in development.

**Comment by macobo** [comment]
:+1: from me

**Pull request merged**